src/main_isobands.js

```diff
@@ -0,0 +1,6 @@
+
+import isoBands from './isobands.js';
+
+export {
```

Comment:
~`export * from './isobands';` should be enough, no import required~

**EDIT**: you don't need this file (see above comments)

Reply:
got rid of it in my development branch! thanks for pointing that out!